producer/ulcl_procedure.go

```diff
@@ -122,7 +120,7 @@ func EstablishPSA2(smContext *context.SMContext) {
 		if nodeAfterULCL {
 			addr := net.UDPAddr{
 				IP:   curDataPathNode.UPF.NodeID.NodeIdValue,
-				Port: pfcpUdp.PFCP_PORT,
+				Port: 8805,
```

Comment:
What about using the one from the `factory`?
```suggestion
				Port: factory.DEFAULT_PFCP_PORT,
```